projects/opendr_ws_2/src/opendr_perception/opendr_perception/panoptic_segmentation_efficient_ps.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+from pathlib import Path
+import argparse
+from typing import Optional
+
+import rclpy
+from rclpy.node import Node
+import matplotlib
+from sensor_msgs.msg import Image as ROS_Image
+
+from opendr_ros2_bridge.opendr_ros2_bridge.bridge import ROS2Bridge
```

Comment:
This line produces `ModuleNotFoundError: No module named 'opendr_ros2_bridge.opendr_ros2_bridge'`

With the following suggestion the error is resolved:

```suggestion
from opendr_ros2_bridge import ROS2Bridge
```